src/pages/Food.jsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Header from '../components/Header';
 import SearchBar from '../components/SearchBar';
 
 function Food({ match }) {
   return (
     <div>
+      <Header title="Comidas" match={ match } />;
       <button type="button" data-testid="search-top-btn">click</button>
```

Comment:
```suggestion
      <Header title="Comidas" match={ match } />
```
A mesma questão do botão click aqui